src/main/resources/config.yml

```diff
@@ -930,6 +930,24 @@ specials:
     # Its not recommended to use values above 1.0
     explosion-factor: 1.0
   
+  # arrow-blocker settings
+  arrow-blocker:
+    # Protection time
+    # type: int
+    # The time to protect the player from projectils
+    protection-time: 10
+    
+    # Protection time
+    # type: int
+    # Time to wait for the next useage
+    using-wait-time: 5
+    
+    # Interact item
+    # type: int or string
+    # The item id or string which indicates which item should be in the
+    # hand to execute the arrow-blocker on interaction
+    item: barrier
```

Comment:
`item: ender_eye`
We need to make sure to block the normal behavior of an Eye of Ender!!